### DISCUSSION TITLE
@motion/solid size is too huge. (18kb)

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [motiondivision/motionone#160] [Support]
![image](https://user-images.githubusercontent.com/87828192/207818747-d13c34d2-49e3-4676-880c-672191c47028.png)

I started to use motion one solid after seeing 
'Motion One's animate function is just 3.8kb, and scroll is only 2.5kb.' on website.
But @motion/solid is 18kb while 'solid-transition-group' is 2.3kb
But I still want to use @motion/solid, it's much smooth.
Is there no way to reduce the @motion/solid size under 5kb?
Please reduce bundle size.....it's too huge. it's not same the website explain...

**Comment by mattgperry** [comment]
We quote gzipped sizes as this is what's served to users. 7kb sounds a little higher than I'd imagine but not much - this is probably the Solid APIs we call. But it isn't just an `animate` function, it has all the state management of which values to animate to, plus the inView/hover/press gestures. If you want just the `animate` bundlesize then you have to import that from `motion` and wire it up yourself.
